api/x509cert.go

```diff
@@ -56,13 +56,20 @@ func (s *SigningService) GetX509CACertificate(ctx context.Context, keyMeta *prot
 		return nil, status.Errorf(codes.InvalidArgument, "Bad request: %v", err)
```

Comment:
Currently in this PR, the changes are for x509 cert only. If we see no issues, I will make similar changes for other cert types.